### PR TITLE
improve dev UX by removing unnecessary toil

### PR DESCRIPTION
**Pull request by sthaha**
* removes unnecessary rebases due to bundle generation
* Adds make target for installing oc
* Removes (ab)use of [[ $(command -v ...) ]] with command -v

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -265,6 +265,10 @@ operator-sdk: ## Download operator-sdk locally if necessary.
 yq: ## Download yq locally if necessary
 	@./hack/tools.sh yq
 
+.PHONY: oc
+oc: ## Download yq locally if necessary
```
Comment (vimalk78) [inline Makefile:269]: comment says `yq`

File: hack/bundle.sh [hunk 2]
```diff
@@ -101,6 +101,14 @@ main() {
 		--select-optional name=operatorhub \
 		--optional-values=k8s-version=1.25 \
 		--select-optional suite=operatorframework
+
+	local csv_file=bundle/manifests/kepler-operator.clusterserviceversion.yaml
+
+	if git diff --ignore-matching-lines='createdAt:' --exit-code "$csv_file" >/dev/null; then
+		info "no changes to $(basename $csv_file) detected; resetting it"
+		run git checkout -- "$csv_file"
```
Comment (vimalk78) [inline hack/bundle.sh:109]: is it safe to do `git checkout` from script

Comment (sthaha) [reply hack/bundle.sh:109]: yeah, the git diff explicitly checks if the only change is in createdAt ..

File: Makefile (hunk 1 above)
Comment (sthaha) [reply Makefile]: Good catch :)

**Review by vimalk78 — APPROVED** [review]

File: hack/bundle.sh (hunk 2 above)
Comment (sthaha) [reply hack/bundle.sh:109]: hehe - https://github.com/operator-framework/operator-sdk/issues/6285#issuecomment-1415350333

**Pull request merged**